Ago-Dic-2020/flores-fernandez-fernando/Practicas/Practica-2/Practica-2.py

```diff
@@ -0,0 +1,59 @@
+#########################################################################
+#Clases
+class Dunedain:
+    def __init__(self,nombre,ocupacion,hp,sp):
+        self.nombre = nombre
+        self.ocupacion = ocupacion
+        self.hp = hp
+        self.vivo= True
+        self.sp=sp
+    
+    def Saludar(self):
```

Comment:
Los nombres de funciones y variables en python deben de ser en minúsculas preferentemente, de acuerdo al estándar oficial. Te recomiendo darle un vistazo 😉  https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Este cambio muy posiblemente te requiera actualizar tu diagrama de clase, así que te olvides de ello 😄

Reply:
sip, cambie los espacios